addons/hive.yml

```diff
@@ -0,0 +1,11 @@
+---
+- name: install hive
+  hosts: frontendnodes
+  become: yes
+  roles:
+    - role: java
+    - role: mysql
+      when: HIVE_METASTORE == 'mysql'
```

Comment:
I wrote this with an option to extend this code to provide support for PostgreSQL in the future if needed. At this point the `HIVE_METASTORE` variable is set using group_vars. Once we add the support for other databases, a user can change the value of this variable to call the corresponding module to provision the metastore(either MySQL or PostgreSQL or Oracle etc.). So shall I keep it the way it is now?